src/Util/PieVersion.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Php\Pie\Util;
+
+use Composer\InstalledVersions;
+
+use function class_exists;
+
+/** @internal This is not public API for PIE, so should not be depended upon unless you accept the risk of BC breaks */
+final class PieVersion
+{
+    /**
+     * Note: magic constant that causes Symfony Console to not display a version
+     * {@see Application::getLongVersion()}
```

Comment:
This is a dead reference, because `Application` is not imported here.

Reply:
Hah, good spot - indeed it's not fully qualified, but PhpStorm doesn't seem to mind, I'd actually checked I could click through, so missed it wasn't fully qualified

![Screenshot from 2024-11-26 09-24-37](https://github.com/user-attachments/assets/a6b2fc55-4d09-49b4-aef4-76d19cdab832)

Reply:
@godruoyi I actually added a comment to the class in your PR:

https://github.com/php/pie/blob/75dbc227ce6445fab7b40bc8f99fdd4ca76ebbcc/src/Util/PieVersion.php#L29-L41